pkg/controller/user-cluster-controller-manager/resources/resources/gatekeeper/webhook.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2020 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gatekeeper
+
+import (
+	"crypto/x509"
+	"fmt"
+
+	admissionregistrationv1beta1 "k8s.io/api/admissionregistration/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/utils/pointer"
+
+	"k8c.io/kubermatic/v2/pkg/resources"
+	"k8c.io/kubermatic/v2/pkg/resources/certificates/triple"
+	"k8c.io/kubermatic/v2/pkg/resources/reconciling"
+)
+
+// ValidatingwebhookConfigurationCreator returns the ValidatingwebhookConfiguration for gatekeeper
+func ValidatingWebhookConfigurationCreator(caCert *x509.Certificate, namespace string) reconciling.NamedValidatingWebhookConfigurationCreatorGetter {
+	return func() (string, reconciling.ValidatingWebhookConfigurationCreator) {
+		return resources.GatekeeperValidatingWebhookConfigurationName, func(validatingWebhookConfigurationWebhookConfiguration *admissionregistrationv1beta1.ValidatingWebhookConfiguration) (*admissionregistrationv1beta1.ValidatingWebhookConfiguration, error) {
+			failurePolicyIgnore := admissionregistrationv1beta1.Ignore
```

Comment:
I changed it because of this issue: https://github.com/kubermatic/kubermatic/issues/5713#issuecomment-666573791 . 
When set to fail, when the validating webhook is malfuncioning it breaks our clusters.

Reply:
I see, what I would suggest is the following. If the webhook is not working as expected, we should offer a mechanism to simply deactivate the webhook and remove it from the user cluster. Because using this policy at this point is kinda confusing. If I enable gatekeeper, then I am expecting some actions and if those actions are not satisfying my scenario(e.g: break my cluster for example) I should be able to simply deactivate it. Another approach can be, to make the policy flexible.

Reply:
Right, added a ticket for this - https://github.com/kubermatic/kubermatic/issues/6008. 

There should be a way to do it differently and I try to find a way before 2.16 rolls out.